src/ModuleResolver.ts

```diff
@@ -334,6 +334,10 @@ export class ModuleResolver implements Disposable {
         if (fs.existsSync(path.join(dir, "node_modules", pkgName))) {
           return dir;
         }
+
+        if (fs.existsSync(path.join(dir, ".prettier-vscode-root"))) {
+          return findUp.stop;
+        }
```

Comment:
I'm not sure this would ever really be used outside of testing, so I didn't put it in the CHANGELOG. This just prevents the ModuleResolver logic from continuing to scan any parent directories if it's found, otherwise the tests would always fallback to the `prettier` definition in the root `package.json`, rather than the explicit defaulting logic.

Right now it's exclusively used for the `no-dep` case where it's required, but you think it makes sense, we may want to see about putting it in each of the workspace roots (or just in the `test-fixtures` directory itself?) as a follow up PR, to help ensure each workspace is testing against the version we expect to be.